Source: oahong@gmail.com
Upstream: NO
Reason: strrchr return type should be const char* in cplusplus
--- pcmanx-gtk2-0.3.8/src/view/telnetview.cpp~	2009-08-07 22:24:55.729503033 +0800
+++ pcmanx-gtk2-0.3.8/src/view/telnetview.cpp	2009-08-07 22:34:43.708503653 +0800
@@ -643,7 +643,7 @@
 #ifdef USE_WGET
 	if (m_bWgetFiles == true) {
 		const char* t_pcURL = sURL.c_str();
-		char* t_pcDot = strrchr(t_pcURL, '.') + 1;
+		const char* t_pcDot = strrchr(t_pcURL, '.') + 1;
 		char t_cFileType = strlen(t_pcURL) - (t_pcDot -t_pcURL);
 		if (t_cFileType == 3) {
 			if (strncmp(t_pcDot, "rar", 3) == 0 ||
